Improving Documentation (#175)
Impostor/Impostor: docs/Running-the-server.md

2. Find the [latest dev release](https://ci.appveyor.com/project/Impostor/Impostor/branch/dev/artifacts).
3. Download either the Windows or the Linux version.
4. Extract the zip.

> 5. Modify `config.json` according to your needs. Docs can be found [here](Server-configuration.md) *(this step is mandatory to whoever wants to expose this server to other machines)*

Reviewer: ```suggestion
5. Modify `config.json` to your liking. Documentation can be found [here](Server-configuration.md) *(this step is mandatory if you want to expose this server to other devices)*
```
